source/bits/ashiftl.ts

```diff
@@ -0,0 +1,19 @@
+import { Bits } from '../types'
+
+/**
+ * Arithmetic Shift Left
+ *
+ * @example
+ * ashiftl([1,0,1,1,0,1]) => [1,1,0,1,1,0]
```

Comment:
This looks very similar to the circular shift to me. Why is having both necessary? Am I missing a use/edge-case?

Reply:
Considering Wikipedia:

With Arithmetic Shift it's always a zero and with circular Shift it can be a one or a zero

Arithmetic Shift Left:
<img width="328" alt="Bildschirmfoto 2020-10-13 um 19 32 01" src="https://user-images.githubusercontent.com/26602940/95895345-d2e80100-0d8a-11eb-8a3e-1303e4aadc0e.png">

Circular Shift Left:
<img width="326" alt="Bildschirmfoto 2020-10-13 um 19 32 08" src="https://user-images.githubusercontent.com/26602940/95895348-d4192e00-0d8a-11eb-8b47-6fc106cf1685.png">